### ISSUE TITLE
Add LSTM to ngen extern

### ISSUE THREAD

**Issue by arpita0911patel** [CIROH-UA/ngen#1]
https://github.com/CIROH-UA/lstm

**Comment by JoshCu** [comment]
added in [add_lstm_extern ](https://github.com/CIROH-UA/ngen/tree/add_lstm_extern) which was merged into #2 too. 

adding to ngiab is just these two lines in the final image stage 
```docker
COPY --from=ngen_clone /ngen/ngen/exterm/lstm /ngen/ngen/extern/lstm
RUN uv pip install --no-cache-dir /ngen/ngen/extern/lstm --extra-index-url https://download.pytorch.org/whl/cpu
``` 
the extra index line install cpu only pytorch which reduces the image size by ~5Gb

**Comment by JoshCu** [comment]
now also in the ngiab branch which will be used for nextgen in a box from now on
